pygsti/objects/spamvec.py

```diff
@@ -2452,6 +2452,7 @@ def from_vector(self, v, close=False, dirty_value=True):
         """
         if self._prep_or_effect == "prep":
             for sv in self.factors:
+                print("Have gpindices: ", sv.gpindices, " v: ", v)
```

Comment:
Is this just a debug statement?  Maybe remove it?

Reply:
Nice catch, removed.